apps/docs/src/app/ui/ui.component.html

```diff
@@ -17,20 +17,61 @@
       </flowbite-navbar-brand>
     </div>
     <flowbite-navbar-content>
-      <flowbite-navbar-item routerLink="./docs">Docs</flowbite-navbar-item>
-      <flowbite-navbar-item>
-        <a
-          href="https://flowbite.com"
-          target="_blank"
-          rel="noopener">
-          Flowbite
-        </a>
-      </flowbite-navbar-item>
+      <div class="flex flex-row items-center gap-5">
```

Comment:
The icons that need to be displayed even if the screen size is tighthave to be outside the navbar content, otherwise it's hidden

Reply:
Have changed it to what I think makes sense, but we'll revisit in future when we can get a clearer answer

Reply:
Waiting for future commit (I sent you some on discord) and it's okay